package.json

```diff
@@ -4,10 +4,10 @@
   "license": "MIT",
   "scripts": {
     "ng": "ng",
-    "start": "npm run serve",
+    "start": "yarn run serve",
```

Comment:
Nitpick: I don't think it's required, but standard project and examples usually come with the following script in order to enforce users to use yarn when they try npm:
```json
    "preinstall": "npx only-allow yarn",
```

Reply:
Nice!